protocols/kad/CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
-## 0.44.3 - unreleased
+## 0.44.4 - unreleased
```

Comment:
You will also need to bump the version in `Cargo.toml` please! :)

Reply:
Sure! Done.